inox2d/src/node/components/drawable.rs

```diff
@@ -0,0 +1,83 @@
+use glam::Vec3;
+
+use super::super::InoxNodeUuid;
+
+/// If has this as a component, the node should render something
+pub struct Drawable {
+	pub blending: Blending,
+	/// If Some, the node should consider masking when rendering
+	pub masks: Option<Masks>,
+}
+
+pub struct Blending {
+	pub mode: BlendMode,
+	pub tint: Vec3,
+	pub screen_tint: Vec3,
+	pub opacity: f32,
+}
+
+#[derive(Default, PartialEq, Clone, Copy)]
+pub enum BlendMode {
+	/// Normal blending mode.
+	#[default]
+	Normal,
+	/// Multiply blending mode.
+	Multiply,
+	/// Color Dodge.
+	ColorDodge,
+	/// Linear Dodge.
+	LinearDodge,
+	/// Screen.
+	Screen,
+	/// Clip to Lower.
+	/// Special blending mode that clips the drawable
+	/// to a lower rendered area.
+	ClipToLower,
+	/// Slice from Lower.
+	/// Special blending mode that slices the drawable
+	/// via a lower rendered area.
+	/// (Basically inverse ClipToLower.)
+	SliceFromLower,
+}
+
+impl BlendMode {
+	pub const VALUES: [BlendMode; 7] = [
+		BlendMode::Normal,
+		BlendMode::Multiply,
+		BlendMode::ColorDodge,
+		BlendMode::LinearDodge,
+		BlendMode::Screen,
+		BlendMode::ClipToLower,
+		BlendMode::SliceFromLower,
+	];
+}
```

Comment:
Why is this here? It's not even used :v
(my guess is you intended to use it for something but forgot to remove it)

Reply:
Fun fact: this is used by the WGPU renderer for some reason. Tho the WGPU renderer is seriously out of date now...

Reply:
Let's keep it for now until we revive the WGPU renderer then. I'll probably move that to inox2d-wgpu eventually